### DISCUSSION TITLE
Searching for a Group for Class Assignments

### DISCUSSION THREAD

**Discussion by admontalvo** [comptoolsres/comptoolsres.github.io#13] [Need a Group]
Good Morning Everyone,

Out of curiosity, does anybody not have a group or has room for one more group member? I am searching for a group to join.

**Comment by rjgott** [comment]
I have not found a group yet and would be happy to work with you.

**Reply by admontalvo** [reply]
That could work. What would be the best way to get in contact with you?

**Reply by rjgott** [reply]
I will send you a message through the course page where we can figure it out.

**Reply by sadie0733** [reply]
I also dont have a group yet.
